### ISSUE TITLE
Expand card tile controls

### ISSUE THREAD

**Issue by alessiocali** [alessiocali/card-tiles#6]
Due to external dependencies we can only click and cycle through faces. It would be nice to have the following options instead:

* Click: iterates faces (not back).
* Double click: flip card (show back face).
* Middle click / Right Click + Menu: recall (delete tile and put card back to its original stack).
* Shift + Click: show original card.

Others to consider:
* Buttons to do the above instead of clicking keybinds.

**Comment by alessiocali** [comment]
Is blocked by #5 .
